docs-site/docs/development/integration/ga4gh_wes_sapporo_setup.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1
+id: ga4gh_wes_sapporo_setup
+title: GA4GH WES Sapporo Setup
+---
+
+## Prerequisites
+
+You will need to have [Docker](https://docs.docker.com/get-docker/) installed for managing Sapporo
+
+## How to set up a Sapporo GA4GH WES instance for development
+
+Note: This should only be used for development purposes, use a production WSGI server for production environments.
+
+### Configure IRIDA Next
+
+Set active storage service to `:local` in `config/environments/development.rb`
+
+```ruby
+config.active_storage.service = :local
+```
+
+Configure environment developer credentials
+
+```bash
+EDITOR="vim --nofork" bin/rails credentials:edit --environment development
+```
+
+```yml
+ga4gh_wes:
+  server_url_endpoint: 'http://localhost:1122/'
+```
+
+### Setup Sapporo (WES implementation)
+
+Download and run the [PHAC-NML Sapporo](https://github.com/phac-nml/sapporo-service) fork in dev docker mode
+
+```bash
+# Clone and checkout the irida-next branch
```

Comment:
**question:** Do I do this in my home directory?

Reply:
Clarification added in https://github.com/phac-nml/irida-next/pull/538/commits/71bfb20d95ffc5c382b1ffca5f2f217d745d2fb8